assist/README.md

```diff
@@ -1,8 +1,141 @@
 RACK ASSIST
 ===========
 
+# Introducing RACK ASSIST
+
 (Automated Software Structure Ingestion and System Testing)
 
+The ASSIST toolkit contains several tools that can be used with RACK.
+
+ * ASSIST-DC  -- Data Collection
+ * ASSIST-DI  -- Data Ingestion
+ * ASSIST-DV  -- Data Verification
+ * ASSIST-SADL  -- SADL translation
+ * ASSIST-RACKLIB -- RACK library
+
+## ASSIST-DC  -- Data Collection
+
+  This portion of the toolkit represents a case study in the
+  integration of ARCOS data collection into the development process
+  that might be utilized by a software development contractor
+  providing input to ARCOS.  The tools here are designed to integrate
+  with a typical development process and gather information related to
+  the build as the build is being performed, including:
+
+    - Who performed the build?
+    - When was the build performed?
+    - What build tools were used (compiler, linker, etc.)?
+    - What output files were generated from what input files, via what activity?
+    - What versions of the source files were used?
+
+  It should be noted that the ASSIST-DC tools are a POC and represent
+  one way to capture a typical build process.  The ASSIST-DC does not
+  represent the only way to capture this data, and it has not been
+  used with ARCOS program-provided input (which is not buildable).
+
+  These tools are used by adding the `assist/databin` directory to the
+  current `PATH` before performing development processes; no
+  additional work is needed to enable ASSIST-DC data collection.  The
+  information collection can be enhanced easily by adding new
+  collectors to this directory.
+
+  The intent of the ASSIST-DC set of tools is to model how a
+  development process might be captured for RACK in a deployed
+  program.
+
+
+## ASSIST-DI  -- Data Ingestion
+
+  The ASSIST-DC subset of tools provides a set of tools that are used
+  to generate local data during the build process.  That local data
+  exists only in the development workspace, and the ASSIST-DI tools
+  are used to ingest that collected data and upload it into a RACK store.
+
+  The ASSIST-DI utilizes a flexible data recognition system and logic
+  programming techniques (written in Prolog) to automatically generate
+  correlations and relationships in the RACK data based on the defined
+  ontology and the data extracted from the artifacts being described.
+  This extracted data can be in any number of forms (the canonical
+  example is the ASSIST-DC-generated data), and ASSIST-DI provides the
+  ability to recognize those input forms and convert them with proper
+  ontology-based relationships for either direct upload to RACK or
+  output as OWL files that can later be uploaded to RACK.
+
+  One particular advantage of ASSIST-DI is that it does not need
+  explicit code updates for changes in the ontology.  There are a
```

Comment:
I think this sentence would be easier to understand if we substituted "when there are" for "for".

Reply:
Thanks, reworded.